cli/determined_cli/shell.py

```diff
@@ -76,8 +76,10 @@ def _open_shell(shell: Command, username: str, additional_opts: str) -> None:
         fp.flush()
         check_len(shell.addresses, 1, "Cannot find address for shell")
 
-        _, port = shell.addresses[0]["host_ip"], shell.addresses[0]["host_port"]
-        host = "localhost"
+        host, port = shell.addresses[0]["host_ip"], shell.addresses[0]["host_port"]
+        if host == "[::1]":
```

Comment:
What is this string we are matching? It seems like it would be better if we created a constant variable that described this value and used it here.

Reply:
"[::1]" is the loopback address and is typically resolved to "localhost". Will fix it using a constant value.